mdal/mdal_utils.hpp

```diff
@@ -64,5 +76,15 @@ namespace MDAL
     return ltrim( rtrim( s, delimiters ), delimiters );
   }
 
+
+  // datetime
+  class DateTime
+  {
+
+  };
```

Comment:
Hmm?

Reply:
smart replacement of QDateTime :dancing_men: